README.md

```diff
@@ -35,6 +37,27 @@ Example:
   <li>Beta: [[orientation.beta]]</li>
   <li>Gamma: [[orientation.gamma]]</li>
 </ul>
+
+<device-motion
+    api-supported="{{apiSupported}}"
+    sensors-available="{{sensorsAvailable}}"
```

Comment:
`{{apiSupported}}` and `{{sensorsAvailable}}` names already used in `<device-orientation>`'s `api-supported` and `sensors-available` properties in lines 27 and 28. Should we use different names, e.g. `{{orientationApiSupported}}`, `{{orientationSensorsAvailable}}` and `{{motionApiSupported}}`, `{{motionSensorsAvailable}}` to avoid data binding conflict (value overriding)?